.github/workflows/ci.yml

```diff
@@ -92,6 +92,9 @@ jobs:
       - run: yarn install --ignore-engines
         # You can run `yarn fix-lint` to fix all Prettier complaints.
       - run: yarn lint
+        # Make sure no tests are skipped with "focused" tests.
+      - run: |
+          ! find tensorboard/webapp -type f -regex '.*_test.ts$' -exec grep -E 'f(it|describe)\(' {} +
```

Comment:
This is correct and works fine, but using `git grep` is clearer:

```
! git grep -E 'f(it|describe)\(' 'tensorboard/webapp/*_test.ts'
```

(It’s also often faster, for what it’s worth.)

Reply:
@stephanwlee , if you go with @wchargin's suggestion, please consider including *_test.ts files under tensorboard/plugins as well.